client/src/utils.js

```diff
@@ -0,0 +1,9 @@
+import moment from 'moment';
+
+export const apiUrl = 'http://localhost:1012';
```

Comment:
imo `apiUrl` isn't a util... I consider utils as functions... I usually create another file called `const.js`, which will hold all my `export const` stuff.

This really depends on each person.